src/providers/BackgroundReplacementProvider/index.tsx

```diff
@@ -179,10 +182,26 @@ export const BackgroundReplacementProvider: FC<
     }
   };
 
+  const changeBackgroundReplacementImage = async (
+    imageBlob: Blob
+  ) => {
+    if (!backgroundReplacementProcessor) {
+      logger.info(`BackgroundReplacementProcessor has not been initialized yet.`);
+      return;
+    }
+    try {
+      await backgroundReplacementProcessor.setImageBlob(imageBlob);
+      logger.info(`Background replacement image changed to new imageBlob: ${imageBlob}`);
+    } catch (error) {
+      logger.error(`Failed to change the background replacement image: ${error}`);
```

Comment:
Under what condition this can fail? should the consumer be notified?

Reply:
If in the js library, it fails to load image with the blob https://github.com/aws/amazon-chime-sdk-js/blob/bec2e70f2488658645494d01c141a6fb2939fa47/src/backgroundreplacementprocessor/BackgroundReplacementFilter.ts#L116, then it will fail. 
By notifying the consumer, do you mean throw an error?

Reply:
Yes. Also do we have a follow up task to update the demo and docs?

Reply:
Are there any restrictions on the image? If yes, let's also make sure the doc covers that.

Reply:
Yes, we have a follow up, there is no restriction if you pass a Blob, and we will put the example in demo and the doc.